sdk/search/search-documents/src/synonymMapHelper.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SynonymMap } from "./serviceModels";
+import { promisify } from "util";
+import * as fs from "fs";
+declare var window: Window & typeof globalThis;
+const readFileAsync = promisify(fs.readFile);
+
+/**
+ * Helper method to create a SynonymMap object. This is a NodeJS only method.
+ *
+ * @param name - Name of the SynonymMap.
+ * @param filePath - Path of the file that contains the Synonyms (seperated by new lines)
+ * @returns SynonymMap object
+ */
+export async function createSynonymMapFromFile(
+  name: string,
+  filePath: string
+): Promise<SynonymMap> {
+  if (isBrowser()) {
```

Comment:
instead of doing this, can we create a `synonymMapHelper.browser.ts` with a dummy implementation that throws?

Reply:
Done